organisations/relationship_transfer.go

```diff
@@ -68,20 +68,31 @@ func getNodeRelationshipNames(cypherRunner neoutils.CypherRunner, uuid string) (
 	return relationshipsFromNodeWithUUID, relationshipsToNodeWithUUID, nil
 }
 
+func constructTransferRelationshipsWithPlatformVersionFromNodeQuery(fromUUID string, toUUID string, predicate string) *neoism.CypherQuery {
+	transferAnnotationsQuery := &neoism.CypherQuery{
+		Statement: fmt.Sprintf(`MATCH (oldNode:Organisation {uuid:{fromUUID}})-[oldRel:%s]->(p)
+					WHERE EXISTS(oldRel.platformVersion)
+					MATCH (newNode:Organisation {uuid:{toUUID}})
+					MERGE (newNode)-[newRel:%s{platformVersion:oldRel.platformVersion}]->(p)
+					on create SET newRel = oldRel
```

Comment:
Should we consider doing this manually for now for problematic orgs and implement this properly when we get to people concordance?